test.ts

```
@@ -0,0 +1,57 @@
+
+  private collectData = () => {
+    // 定期收集数据但从不清理
+    const largeData = new Array(5000).fill(0).map(() => new Uint8Array(1024));
+    this.resources.set(`data_${Date.now()}`, largeData);
+  };
+
+  // 缺少析构函数或清理方法来移除事件监听器和释放资源
+}
+
+// 创建实例但从不销毁
+let manager: ResourceManager | null = new ResourceManager();
```

ESLint

[error] 'manager' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)